source/mainnet/net/resources/sirius-testnet-reset-wallets.rst

```diff
@@ -14,4 +14,4 @@ Go to :ref:`Mobile Wallet downloads<downloads-mobile-wallet-testnet>` to get the
 Desktop Wallet
 ==============
 
-Need instructions for how to get rid of data after uninstall. Then reinstall the correct version for your platform. Go to :ref:`Desktop Wallet downloads<downloads-desktop-wallet-testnet>` to get the file for your platform.
+To clear the data of the wallet, uninstall it and follow :ref:`the instructions for resetting the wallet's data` (Remember to remove the testnet and *not* the mainnet files). Then reinstall the correct version for your platform. Go to :ref:`Desktop Wallet downloads<downloads-desktop-wallet-testnet>` to get the file for your platform.
```

Comment:
It is not necessary to reinstall the desktop wallet after deleting the user data, yet we ask them to do so here. Is there a reason for that?

Reply:
Because that is what was mentioned in the meeting yesterday about documentation for the testnet reset. No one seemed sure if it would be which is why we asked DW experts to be involved.